addons/l10n_mx/models/chart_template.py

```diff
@@ -17,6 +17,8 @@ def _load_template(
             company, code_digits=code_digits,
             transfer_account_id=transfer_account_id, account_ref=account_ref,
             taxes_ref=taxes_ref)
+        if not self.env.user.company_id.country_id == self.env.ref('base.mx'):
```

Comment:
It is a bad practice to do that, a lot of things depends of the company of the user, you can not install the localization like that.

I do not like check the module I prefer the country itself. I will think on that BTW.